pkg/iostreams/iostreams.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/google/shlex"
 	"github.com/mattn/go-colorable"
 	"github.com/mattn/go-isatty"
+	"github.com/schollz/progressbar/v3"
```

Comment:
While the briandowns/spinner library used in `progessIndicator` has some progress bar styles, it doesn't seem to support creating a progress bar that is incrementally updated as we work towards completion of a task (in this case, downloading n release assets)

Reply:
As we are just updating the spinner as each asset is downloaded, would an alternative solution be to continue to use the `briandowns/spinner` library but add annotations to the spinner as each asset is download to indicate to the user that progress is being made? 

For example:
```bash
:spinner: Downloading asset 1
:spinner: Downloading asset 2
:spinner: Downloading asset 3
...
```

Reply:
Yes, definitely. If it would be preferable to stick with the existing spinner library, I can reconfigure this to avoid introducing a new dependency.

Reply:
Hi @samcoe - I just pushed a new commit with a simplified approach - stuck with the existing spinner, and updated the label with each new asset we start downloading. Not as precise, but still helps to see if progress is being made. I experimented with each asset getting its own progress indicator / line in the output, but it got noisy once I started downloading more than ~10 assets. Let me know what you think!

![demo](https://user-images.githubusercontent.com/30248392/236051122-981a1bb8-54c0-4fb5-a8b4-06ef811e24fe.gif)